src/uml/requestHandling.puml

```diff
@@ -0,0 +1,22 @@
+@startuml
+!include clean.skin
+"SQL Client" -> "Exasol Core" : Select Request
+activate "Exasol Core"
+"Exasol Core" -> "Exasol Core": parse
+"Exasol Core" -> "Virtual Schema Adapter": getCapabilities
+activate "Virtual Schema Adapter"
+"Exasol Core" <<-- "Virtual Schema Adapter": list of capabilities
+deactivate "Virtual Schema Adapter"
+"Exasol Core" -> "Exasol Core": rewrite to remove unsupported parts
+"Exasol Core" -> "Virtual Schema Adapter": pushdown("SELECT ...")
+activate "Virtual Schema Adapter"
+"Exasol Core" <<-- "Virtual Schema Adapter": "IMPORT FROM JDBC ..."
```

Comment:
Import from JDBC is only one example, we can also use IMPORT INTO table, SELECT * FROM VALUESetc.
Maybe you should mention that it's only an example